ios/ios_recommended_tools.md

```diff
@@ -2,4 +2,7 @@
 Useful tools recommended for every iOS developer to use.
 
 ##Breakpoints
-[Useful collection of Xcode breakpoints](https://gist.github.com/gbernady/1fc53d70587b9064f84c)
+Breakpoints are helpful for debugging code. Specifically if a line of code was called when you expect it to be called or not. Breakpoints can be added, moved, edited, and deleted while debugging and don't require a recompile.
```

Comment:
My fault on not being specific before. This section explaining breakpoints themselves can be removed. I think most everyone knows what breakpoints are and how to generally use them.

I think the article from Natasha on _user breakpoints_ is useful to keep in though.